src/System.Net.NameResolution/tests/FunctionalTests/TestSettings.cs

```diff
@@ -17,6 +17,9 @@ internal static class TestSettings
 
         public const string LocalIPString = "127.0.0.1";
 
+        // Timeout values in milliseconds.
+        public const int PassingTestTimeout = 10000;
```

Comment:
Is this sufficiently long?  I could imagine in CI with lots of network traffic and congestion that DNS requests could take longer than this every now and then. I think we use 30 seconds in similar places elsewhere.

Reply:
Changed to 30_000